analysis/2026-05-15-regression/n5-sweep.md

```
@@ -0,0 +1,51 @@
+
+**Baseline** (`cf6ddb7`, 2026-04-30) — 5 iters  
+**Current**  (`1b61fe3`, 2026-05-15) — 5 iters
+
+## Pass rate
+- Baseline: 5/5
+- Current:  5/5
+
+## Per-iteration data
+
+### Baseline (`cf6ddb7`)
+| iter | time (s) | turns | tools | cost ($) | total tk | input tk | output tk | cached tk |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)